src/Uno.UI/UI/Xaml/Controls/TextBlock/TextVisual.Skia.cs

```diff
@@ -18,7 +20,21 @@ internal class TextVisual : Visual
 
 		private static SKTypeface FromFamilyName(string name)
 		{
-			return SKTypeface.FromFamilyName(name);
+			if (name.StartsWith(XamlFilePathHelper.AppXIdentifier))
+			{
+				var uriPath = new Uri(name).PathAndQuery;
```

Comment:
Codacy found an issue: [Remove the unused local variable 'uriPath'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5900308)